lib/screens/inject.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Screens.Inject do
+  @moduledoc "Conveniences for dependency injection and mocking."
+
+  @doc """
+  When the Mix env is not `test`, resolves to the given module name. When it is, resolves to the
+  module name with `.Mock` appended.
+
+  The mock module is not automatically defined (see e.g. `test/support/mocks.ex`).
```

Comment:
I tried for a bit to actually make this the case (auto-create the mock if not already defined), but kept running into issues with "cannot define module ... because it is currently being defined". This may require some higher-level compiler wizardry than I can conjure.